proto/v1/actuator_service.proto

```diff
@@ -12,15 +12,12 @@ option go_package = "generated-go/v1";
 
 service ActuatorService {
   rpc GetActuatorInfo(GetActuatorInfoRequest) returns (ActuatorInfo) {
-    option (google.api.http) = {
-      get: "/v1/actuator/info"
-    };
+    option (google.api.http) = {get: "/v1/actuator/info"};
```

Comment:
Better to pretty format this http option, but it looks like buf format doesn't have this option.